app/csp.py

```diff
@@ -10,6 +10,12 @@ def add_header(response):
     Add a Content Security Policy (CSP) header to the given response.
     '''
 
+    if request.path.startswith(url_for('admin.index')):
+        # Ugh, flask-admin has inline scripts. Since only a handful of
+        # users will have access to this view anyways, just disable CSP
+        # for it.
+        return response
```

Comment:
Filed https://github.com/flask-admin/flask-admin/issues/1135 so we hopefully can remove this someday.